src/dom.js

```diff
@@ -284,7 +284,7 @@ class dom {
 		var limit = event.currentTarget.parentNode;
 		var defFns = [];
 
-		while (currElement && currElement !== limit && !event.stopped) {
+		while (currElement && currElement !== limit && !event.stopped && !currElement.disabled) {
```

Comment:
Awesome, thanks for looking into this.

Since there's this difference between browsers I think it's best if we only ignore clicks, so that the behavior is consistent without us having to have different code for each browser, besides being a more common event too.